aws/solutions/Dynamic-Create-ENIs/Dynamic-Create-ENIs.json

```diff
@@ -0,0 +1,235 @@
+                    "Statement": [
+                        {
+                            "Effect": "Allow",
+                            "Principal": {
+                                "Service": [
+                                    "lambda.amazonaws.com"
+                                ]
+                            },
+                            "Action": [
+                                "sts:AssumeRole"
+                            ]
+                        }
+                    ]
+                },
+                "Path": "/",
+                "Policies": [
+                    {
+                        "PolicyName": "root",
+                        "PolicyDocument": {
+                            "Version": "2012-10-17",
+                            "Statement": [
+                                {
+                                    "Effect": "Allow",
+                                    "Action": [
+                                        "ec2:CreateNetworkInterface",
+                                        "ec2:CreateTags",
+                                        "ec2:DescribeNetworkInterfaces",
+                                        "ec2:DeleteNetworkInterface"
+                                    ],
+                                    "Resource": "*"
+                                },
+                                {
+                                    "Effect": "Allow",
+                                    "Action": [
+                                        "logs:CreateLogGroup",
+                                        "logs:CreateLogStream",
+                                        "logs:PutLogEvents"
+                                    ],
+                                    "Resource": "*"
+                                }
+                            ]
+                        }
+                    }
+                ]
+            }
+        },
+        "IPsTrigger": {
+            "DependsOn": [
+                "CrudIPs"
+            ],
+            "Type": "Custom::IPs",
+            "Properties": {
+                "ServiceToken": {
+                    "Fn::GetAtt": [
+                        "CrudIPs",
+                        "Arn"
+                    ]
+                },
+                "ips": {
+                    "Ref": "ipList"
+                },
+                "subnets": {
+                    "Ref": "subnet"
+                },
+                "securitys": {
+                    "Ref": "security"
+                }
+            }
+        },
+        "CrudIPs": {
+            "Type": "AWS::Lambda::Function",
+            "DeletionPolicy": "Delete",
+            "DependsOn": [
+                "IamRoleLambda"
+            ],
+            "Properties": {
+                "Code": {
+                    "ZipFile": {
+                        "Fn::Join": [
+                            "\n",
+                            [
+                                "var AWS = require('aws-sdk');",
```

Comment:
Given this big chunk of code, this template might be better as yaml to make this bit more readable.

Reply:
Or even better, store the source in a separate file and instruct use of `aws cloudformation package` in the README.